core/domain/param_domain.py

```diff
@@ -26,19 +26,49 @@
 
 class ParamSpec(object):
     """Value object for an exploration parameter specification."""
+
     def __init__(self, obj_type):
+        """Initializes a ParamSpec object with the specified object type
+
+        Args:
+            obj_type: unicode. The object type with which the parameter is
+            initialized.
+        """
         self.obj_type = obj_type
 
+
     def to_dict(self):
+        """Adds the object type to a dict, with the key as `obj_type`
+        and the value as the type of the object eg.(UnicodeString).
+
+        Returns:
+            dict. The key of the dict is `obj_type` and the value is the type
```

Comment:
In general, there's no such thing as "the key of the dict", since dicts can have multiple keys. So, a slight rephrasing:

    dict. A dict with a single key, `obj_type`, whose value is the type of the parameter represented by this ParamSpec.